tests/EpcGeneratorTest.php

```diff
@@ -5,6 +5,7 @@
 use Rexlabs\EpcGenerator\EpcGenerator;
 use org\bovigo\vfs\vfsStream;
 use PHPUnit\Framework\TestCase;
+use org\bovigo\vfs\vfsStreamDirectory;
```

Comment:
Can we fix the order of the 'use' statements, please?
Maybe we should just include the code fix library 🤔

Reply:
I have fixed the order. And not sure what package, and also wondering if we really need it.

Reply:
Nah, don't really needed. Was just a thought 🙂